src/client/components/common/Navbar.jsx

```diff
@@ -23,22 +23,21 @@ class NavigationBar extends Component {
   render() {
     return (
       <Navbar type="light" theme="white" expand="md" className={NavStyle.Main}>
-          <NavbarBrand href="#">
-            <img
-              alt=""
-              src="https://res.cloudinary.com/dtocmcjrg/image/upload/v1575006713/myriade_logo_m9gab5.svg"
-              width="150"
-            /> 
-          </NavbarBrand>
-          <Nav navbar className="ml-auto">
-            <NavItem >
-              
-              <Button pill outline theme='secondary'>
-                <BlackLink to={ROUTES.DASHBOARD}> Dashboard </BlackLink>
-              </Button>
-            </NavItem>
-          </Nav>
-        
+        <NavbarBrand href="#">
+          <img
+            alt=""
+            src="./public/icons/logo_text.svg"
+            height="45"
```

Comment:
I did every task I knew how to solve on #12 except Paul's request to remove the '#' from the URL, and adding the favicon to the tab bar. Here's what I got when I ran **make build**:
```
npm run build

> personal@1.0.0 build /home/jonathan/Documents/Coding/Myriade/web
> webpack --mode production

Browserslist: caniuse-lite is outdated. Please run next command `npm update`
Hash: b6444e91b8bdcbe54eb3
Version: webpack 4.41.2
Time: 27291ms
Built at: 12/02/2019 10:36:00 PM
     Asset      Size  Chunks                    Chunk Names
index.html  1.16 KiB          [emitted]         
   main.js  2.43 MiB       0  [emitted]  [big]  main
Entrypoint main [big] = main.js
 [20] (webpack)/buildin/global.js 472 bytes {0} [built]
 [29] ./src/client/styles/pages/Signup.less 1.27 KiB {0} [built]
[132] ./src/client/styles/pages/Dashboard.less 1.28 KiB {0} [built]
[197] ./src/client/styles/pages/Login.less 1.27 KiB {0} [built]
[198] ./src/client/styles/main.less 1.24 KiB {0} [built]
[287] ./src/client/styles/pages/Landing.less 1.27 KiB {0} [built]
[289] ./node_modules/@emotion/is-prop-valid/dist/is-prop-valid.browser.esm.js + 1 modules 4.38 KiB {0} [built]
      |    2 modules
[290] ./node_modules/merge-anything/dist/index.esm.js + 1 modules 10.4 KiB {0} [built]
      |    2 modules
[414] ./node_modules/css-loader/dist/cjs.js??ref--9-1!./node_modules/less-loader/dist/cjs.js!./src/client/styles/pages/Dashboard.less 789 bytes {0} [built]
[571] ./node_modules/css-loader/dist/cjs.js??ref--9-1!./node_modules/less-loader/dist/cjs.js!./src/client/styles/pages/Login.less 565 bytes {0} [built]
[578] ./node_modules/css-loader/dist/cjs.js??ref--9-1!./node_modules/less-loader/dist/cjs.js!./src/client/styles/pages/Landing.less 268 bytes {0} [built]
[579] ./node_modules/css-loader/dist/cjs.js??ref--9-1!./node_modules/less-loader/dist/cjs.js!./src/client/styles/pages/Signup.less 462 bytes {0} [built]
[581] ./src/client/styles/shards/shards.scss 1.24 KiB {0} [built]
[582] ./node_modules/css-loader/dist/cjs.js!./node_modules/sass-loader/dist/cjs.js!./src/client/styles/shards/shards.scss 94.6 KiB {0} [built]
[583] ./src/client/index.js + 77 modules 2.31 MiB {0} [built]
      | ./src/client/index.js 669 bytes [built]
      | ./src/client/layers/AuthLayer.jsx 4.95 KiB [built]
      | ./src/client/layers/ErrorLayer.jsx 3.65 KiB [built]
      | ./src/client/App.jsx 3.91 KiB [built]
      | ./src/client/utils/routes.js 158 bytes [built]
      | ./src/client/utils/config.js 155 bytes [built]
      | ./src/client/pages/Dashboard.jsx 9.01 KiB [built]
      | ./src/client/pages/LoginPage.jsx 5.72 KiB [built]
      | ./src/client/pages/LandingPage.jsx 2.99 KiB [built]
      | ./src/client/pages/SignupPage.jsx 9.37 KiB [built]
      | ./src/client/utils/graphql.js 415 bytes [built]
      |     + 67 hidden modules
    + 871 hidden modules

WARNING in asset size limit: The following asset(s) exceed the recommended size limit (244 KiB).
This can impact web performance.
Assets: 
  main.js (2.43 MiB)

WARNING in entrypoint size limit: The following entrypoint(s) combined asset size exceeds the recommended limit (244 KiB). This can impact web performance.
Entrypoints:
  main (2.43 MiB)
      main.js


WARNING in webpack performance recommendations: 
You can limit the size of your bundles by using import() or require.ensure to lazy load some parts of your application.
For more info visit https://webpack.js.org/guides/code-splitting/
Child html-webpack-plugin for "index.html":
     1 asset
    Entrypoint undefined = index.html
    [0] ./node_modules/html-webpack-plugin/lib/loader.js!./public/index.html 1.2 KiB {0} [built]```

Is this good to go?

Reply:
Sorry about the cloudinary links, I didn't know about their importance when I was cropping the icons. I've put in the links for the new icons in my latest commit. However, the webpack throws an error when I supply a cloudinary link as a favicon, so I'm still using the .svg file for that.

This PR solves every issue on #12 except the last one involving the Hashrouter.
Here's the error that **make build** threw:
```
WARNING in asset size limit: The following asset(s) exceed the recommended size limit (244 KiB).
This can impact web performance.
Assets: 
  main.js (2.43 MiB)

WARNING in entrypoint size limit: The following entrypoint(s) combined asset size exceeds the recommended limit (244 KiB). This can impact web performance.
Entrypoints:
  main (2.43 MiB)
      main.js


WARNING in webpack performance recommendations: 
You can limit the size of your bundles by using import() or require.ensure to lazy load some parts of your application.
For more info visit https://webpack.js.org/guides/code-splitting/
Child html-webpack-plugin for "index.html":
     1 asset
    Entrypoint undefined = index.html
    [0] ./node_modules/html-webpack-plugin/lib/loader.js!./public/index.html 1.2 KiB {0} [built]
```

Are we good to merge?

Reply:
Yes